tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Wcf/WcfCommon.cs

```diff
@@ -90,12 +91,13 @@ internal static Scope CreateScope<TRequestContext>(TRequestContext requestContex
                 var requestHeaders = requestMessage.Headers;
                 string action = requestHeaders.Action;
                 Uri requestHeadersTo = requestHeaders.To;
+                var cleanedUrl = requestHeadersTo != null ? UriHelpers.CleanUri(requestHeadersTo, removeScheme: false, tryRemoveIds: false) : null;
```

Comment:
Just FYI, if `tryRemoveIds: false`, this won't do the obfuscation you're adding 🙂